src/app/pages/server-details/server-details.component.ts

```diff
@@ -408,7 +408,7 @@ export class ServerDetailsComponent implements OnInit, OnDestroy {
           }
         }
       }).catch((error) => {
-        console.error('Failed to load server data:', error);
+        this.analytics.SentryException(error, {tags: { location: this.constructor.name, function: 'getServers' }});
```

Comment:
I don't think we want to get a sentry exception for all 404 requests

Reply:
Definitely not.